<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="atoum testsuite">
  <testsuite name="SubscribersList" package="Calendar\tests\units" tests="6" failures="0" errors="0" skipped="0" time="0.029141902923584">
    <testcase name="testEmptyInit" time="0.0039298534393311" classname="Calendar\tests\units\SubscribersList" assertions="0"/>
    <testcase name="testNoSubChoice" time="0.0026910305023193" classname="Calendar\tests\units\SubscribersList" assertions="3"/>
    <testcase name="testPopulatedChoice" time="0.0048918724060059" classname="Calendar\tests\units\SubscribersList" assertions="1"/>
    <testcase name="testMultiplePersonChoice" time="0.0051019191741943" classname="Calendar\tests\units\SubscribersList" assertions="2"/>
    <testcase name="testAddSubscribers" time="0.0046131610870361" classname="Calendar\tests\units\SubscribersList" assertions="4"/>
    <testcase name="testAddMultipleSubs" time="0.0079140663146973" classname="Calendar\tests\units\SubscribersList" assertions="2"/>
  </testsuite>
  <testsuite name="Mail" package="Mailing\tests\units" tests="5" failures="0" errors="0" skipped="0" time="0.012051820755005">
    <testcase name="testConstruct" time="0.0022380352020264" classname="Mailing\tests\units\Mail" assertions="0"/>
    <testcase name="testGetSetSender" time="0.0023260116577148" classname="Mailing\tests\units\Mail" assertions="8"/>
    <testcase name="testGetSetRecipient" time="0.002877950668335" classname="Mailing\tests\units\Mail" assertions="2"/>
    <testcase name="testGetSetSubject" time="0.0024058818817139" classname="Mailing\tests\units\Mail" assertions="2"/>
    <testcase name="testGetSetBody" time="0.0022039413452148" classname="Mailing\tests\units\Mail" assertions="2"/>
  </testsuite>
  <testsuite name="MailSender" package="Mailing\tests\units" tests="1" failures="0" errors="0" skipped="0" time="2.3126602172852E-5">
    <testcase name="testSendMail" time="2.3126602172852E-5" classname="Mailing\tests\units\MailSender" assertions="0"/>
  </testsuite>
  <testsuite name="DAO" package="DAO\tests\units" tests="2" failures="0" errors="2" skipped="0" time="0">
    <testcase name="testConnexion" time="0" classname="DAO\tests\units\DAO" assertions="0">
      <error type="UNKNOWN"><![CDATA[Fatal error: Uncaught Error: Class 'DAO\PDO' not found in D:\m2test2\Source\Vendor\DAO\DAO.php:30
Stack trace:
#0 D:\m2test2\Source\Vendor\DAO\DAO.php(12): DAO\DAO->requestConnexion()
#1 D:\m2test2\Test\Vendor\DAO\TestDao.php(12): DAO\DAO->__construct()
#2 phar://D:/m2test2/applis/atoum/atoum.phar/1/classes/test.php(1363): DAO\tests\units\DAO->testConnexion()
#3 -(1): mageekguy\atoum\test->runTestMethod('testConnexion')
#4 {main}
  thrown in D:\m2test2\Source\Vendor\DAO\DAO.php on line 30]]></error>
      <error type="255"><![CDATA[]]></error>
    </testcase>
  </testsuite>
</testsuites>
